### PR TITLE
drds_instance resource supports timeouts setting. 

### PR DESCRIPTION
**Pull request by Lexsss**

### CODE REVIEW

File: alicloud/resource_alicloud_drds_instance.go [hunk 1]
```diff
@@ -125,14 +143,14 @@ func resourceAliCloudDRDSInstanceRead(d *schema.ResourceData, meta interface{})
 	client := meta.(*connectivity.AliyunClient)
 	drdsService := DrdsService{client}
 
-	res, err := drdsService.DescribeDrdsInstance(d.Id())
+	object, err := drdsService.DescribeDrdsInstance(d.Id())
 	if err != nil {
 		if NotFoundError(err) {
 			d.SetId("")
 			return nil
 		}
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_drds_instance.go]: return WrapError(err)

File: alicloud/resource_alicloud_drds_instance_test.go [hunk 2]
```diff
@@ -82,6 +83,184 @@ func testSweepDRDSInstances(region string) error {
...
+					testAccCheck(map[string]string{
+						"description": name,
+					}),
+				),
+			},
+			{
+				ResourceName:      resourceId,
+				ImportState:       true,
+				ImportStateVerify: false,
+			},
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_drds_instance_test.go]: add update description testcase

File: alicloud/resource_alicloud_drds_instance_test.go [hunk 3]
```diff
@@ -82,6 +83,184 @@ func testSweepDRDSInstances(region string) error {
...
+					"instance_series":      "${var.instance_series}",
+					"instance_charge_type": "PostPaid",
+					"vswitch_id":           "${alicloud_vswitch.foo.id}",
+					"specification":        "drds.sn1.4c8g.8C16G",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"description": name,
+					}),
+				),
```
Comment (YY-tomorrow) [inline alicloud/resource_alicloud_drds_instance_test.go]: still

File: alicloud/resource_alicloud_drds_instance_test.go [hunk 4]
```diff
@@ -82,156 +82,192 @@ func testSweepDRDSInstances(region string) error {
...
+		CheckDestroy:  rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"description":          "${var.name}-${count.index}",
+					"zone_id":              "${data.alicloud_zones.default.zones.0.id}",
+					"instance_series":      "${var.instance_series}",
+					"instance_charge_type": "PostPaid",
+					"specification":        "drds.sn1.4c8g.8C16G",
+					"count":                "10",
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_drds_instance_test.go]: Please modify the count to 5

File: alicloud/service_alicloud_drds.go [hunk 5]
```diff
@@ -14,93 +12,75 @@ type DrdsService struct {
 }
 
 // crate Drdsinstance
-func (s *DrdsService) CreateDrdsInstance(req *drds.CreateDrdsInstanceRequest) (response *drds.CreateDrdsInstanceResponse, err error) {
+func (s *DrdsService) CreateDrdsInstance(request *drds.CreateDrdsInstanceRequest) (response *drds.CreateDrdsInstanceResponse, err error) {
```
Comment (xiaozhu36) [inline alicloud/service_alicloud_drds.go]: The create method should not a service method.

File: alicloud/service_alicloud_drds.go [hunk 6]
```diff
@@ -14,93 +12,75 @@ type DrdsService struct {
...
-	return resp, err
-
+	return response, nil
 }
 
-func (s *DrdsService) ModifyDrdsInstanceDescription(request *drds.ModifyDrdsInstanceDescriptionRequest) (response *drds.ModifyDrdsInstanceDescriptionResponse, err error) {
-	req := drds.CreateModifyDrdsInstanceDescriptionRequest()
-	req.DrdsInstanceId = request.DrdsInstanceId
-	req.Description = request.Description
+func (s *DrdsService) RemoveDrdsInstance(id string) (response *drds.RemoveDrdsInstanceResponse, err error) {
```
Comment (xiaozhu36) [inline alicloud/service_alicloud_drds.go]: The delete method should not a service method.

File: website/docs/r/drds_instance.html.markdown [hunk 7]
```diff
@@ -54,6 +54,16 @@ The following arguments are supported:
     - `drds.sn1.32c64g` for DRDS instance Extreme Edition;
         - value range : `drds.sn1.32c64g.128c256g`
        
+### Timeouts
+
+-> **NOTE:** Available in 1.48.0+.
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration-0-11/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 10 mins) Used when creating the drds instance (until it reaches the initial `0` status).
```
Comment (xiaozhu36) [inline website/docs/r/drds_instance.html.markdown]: `(until it reaches running status). `

File: alicloud/resource_alicloud_drds_instance_test.go [hunk 8]
```diff
@@ -82,156 +82,192 @@ func testSweepDRDSInstances(region string) error {
...
+					"vswitch_id":           "${alicloud_vswitch.foo.id}",
+					"specification":        "drds.sn1.4c8g.8C16G",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"description": name,
+					}),
+				),
+			},
+			{
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_drds_instance_test.go]: There also needs to check `Importer`

File: website/docs/r/drds_instance.html.markdown [hunk 9]
```diff
@@ -54,6 +54,16 @@ The following arguments are supported:
     - `drds.sn1.32c64g` for DRDS instance Extreme Edition;
         - value range : `drds.sn1.32c64g.128c256g`
        
+### Timeouts
+
+-> **NOTE:** Available in 1.48.0+.
```
Comment (xiaozhu36) [inline website/docs/r/drds_instance.html.markdown]: 1.49.0

**Pull request merged**